recipes/green/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.11.0" %}
+
+package:
+  name: greem
+  version: {{ version }}
+
+source:
+  fn: green-{{ version }}.tar.gz
+  url:   https://github.com/CleanCut/green/archive/{{version}}.tar.gz
+  sha256: fb78e51a15872c6a4f2952fc6857c91fef390d89b1d725d85b609c119584c137
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
```

Comment:
You should add [entry points](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points) for [the `green` script](https://github.com/CleanCut/green/blob/master/setup.py#L36).